templates/webserver/webserver-deployment.yaml

```diff
@@ -83,10 +83,8 @@ spec:
               mountPath: {{ template "airflow_local_setting_path" . }}
               subPath: airflow_local_settings.py
               readOnly: true
-{{- if .Values.webserver.jwtSigningCertificateSecretName }}
-            - name: signing-certificate
-              mountPath: {{ template "airflow_webserver_jwt_cert_dir" . }}
-              readOnly: true
+{{- if .Values.webserver.extraVolumeMounts }}
```

Comment:
We're still mounting webserver_config.py on line 80, should we be?

Equally for airflow_local_settings.py - that's probably a platform only-feature.

Reply:
We actually have a default `airflow_local_settings.py` baked in that adds `kubernetes-executor` / `kubernetes-pod-operator` labels to dynamic pods, which is kind of useful, and kind of serves as a nice little demo of the feature.

I could see us just removing it and treating it just like we're treating the `webserver_config.py` though. 

Yes I think you are right about the `webserver_config.py`. That should be conditionally added, if one is defined. I'll fix that.

Reply:
fixed these.